.husky/check_unstaged.py

```diff
@@ -4,12 +4,12 @@
 
 
 def print_red(text):
-    return f"\033[91m{text}\033[0m"
+    return "\033[91m{}\033[0m".format(text)
```

Comment:
you dislike python `f`strings?

Reply:
I love them! I was just making the script work in both python2 and 3.

Reply:
ahhhhh that makes sense lol